src/Stratis.Bitcoin/IFullNode.cs

```diff
@@ -49,4 +52,15 @@ public interface IFullNode : IDisposable
         /// <returns></returns>
         T NodeFeature<T>(bool failWithError = false);
     }
+
+    public enum FullNodeState
```

Comment:
This code could be much happier, if there was a description on which state means what and how the node changes from one state to another et cetera

Reply:
Isn't is too obvious to write down? State names are very self explanatory

Reply:
Fixed.